.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] **[REQUIRED CHANGES]** The `test.md` file should be ignored by `.gitignore`, so you can learn how the `.gitignore` works! If you had committed the `test.md` file before the `.gitignore`, it is important that you delete the `test.md` from your repository (try deleting it from your local folder and pushing the changes!)